test/cbmc/proofs/ARP/ARPProcessPacket/ARPProcessPacket_harness.c

```diff
@@ -8,12 +8,14 @@
 #include "FreeRTOS_IP.h"
 #include "FreeRTOS_IP_Private.h"
 #include "FreeRTOS_ARP.h"
+#include "FreeRTOS_Routing.h"
 
 /* This pointer is maintained by the IP-task. Defined in FreeRTOS_IP.c */
 extern NetworkBufferDescriptor_t * pxARPWaitingNetworkBuffer;
 
 /* This is an output function and need not be tested with this proof. */
-void FreeRTOS_OutputARPRequest( uint32_t ulIPAddress )
+void FreeRTOS_OutputARPRequest_Multi( NetworkEndPoint_t * pxEndPoint,
```

Comment:
As we are removing FreeRTOS_OutputARPRequest, don't we have the case where flow is from FreeRTOS_OutputARPRequest to -> FreeRTOS_OutputARPRequest_Multi?